packages/bundler/.mocharc.yaml

```diff
@@ -1,4 +1,4 @@
 timeout: 5000
 require: source-map-support/register
-spec: "dist/test/**/*.js"
+spec: "dist/test/**/*.test.js"
```

Comment:
There's also a glob like this in .vscode/settings.json under mochaExplorer.files. Might be worth updating that too. I could see myself forgetting the suffix and running the tests in the IDE and not catching my mistake. This would also need to be copied to the private repo settings.json.